docs/setup-guide/scripts/host_2_1.sh

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
+
+set -e 
+
+echo -e "\n"
+#To kill running process of cloudcore and edgecore
```

Comment:
Yes, your right. this script is: when you re-run the script, the scrip will check and see if cloudcore or edgecore is runng, it will kill cloudcore or edgecore because cloudecore or edgecore run at background. this just checks and make sure no cloudcore or edgecore runs at backgroud.

Reply:
I make change based on your comment

Reply:
But in the script, it does not catch any cloudcore process. As I mentioned in the line 32, the $cloudcore is always empty

Reply:
I remove the cloudcore check.